features/FEATURE_LWIP/lwip-interface/LWIPInterface.cpp

```diff
@@ -587,6 +587,11 @@ nsapi_error_t LWIP::Interface::bringdown()
 #if LWIP_IPV6
     mbed_lwip_clear_ipv6_addresses(&netif);
 #endif
+#if LWIP_IPV4
```

Comment:
Why was this needed? My recollection was that lwip did zero this itself, and forgot to for IPv6, but maybe that's not true.

Reply:
@andreaslarssonublox could you please respond to this?

Reply:
I think it was not done for IPv4 and discovered for some case like re-connecting with static IP.

Reply:
Okay, I've got no problem with this going in - don't see how it can hurt.

But let's try to avoid too many miscellaneous tweaks going in - we've got a deadline (Friday) and a misbehaving CI system.